cijoe/tests/tools/test_xnvme.py

```diff
@@ -79,10 +79,18 @@ def test_format(cijoe, device, be_opts, cli_args):
 def test_sanitize(cijoe, device, be_opts, cli_args):
     if be_opts["be"] == "linux" and be_opts["admin"] in ["block"]:
         pytest.skip(reason="[admin=block] does not implement sanitize")
+    if "ramdisk" in device["labels"]:
+        pytest.skip(reason="[be=ramdisk] does not implement sanitize")
+
+    pytest.skip(reason="QEMU NVMe devices doesn't support sanitize")
+
+    err, _ = cijoe.run(f"xnvme sanitize --sanact 0x2 {cli_args}")
+
+    assert not err
 
-    pytest.skip(reason="TODO: always fails. Investigate.")
+    # Get log sanitize status log page
+    err, _ = cijoe.run(f"xnvme log-sanitize {cli_args} --nsid {device['nsid']}")
```

Comment:
Is test_sanitize still always skipped? And what if the test is run on non-QEMU devices?

Reply:
Hmm, right perhaps it would be better for the test to need a device with label `sanitize`.